lib/source/encoding.pm

```diff
@@ -0,0 +1,72 @@
+package source::encoding;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.01';
+
+our $ascii_hint_bits = 0x00000010;
```

Comment:
Why isn't this a constant?

Reply:
It could be; whatever I cargo-culted this from didn't have its corresponding variable be a constant.  I don't know how to make it constant without also using Internals::SvREADONLY

Reply:
How about `use constant { ASCII_HINT_BITS => 0x00000010 };`?

Reply:
It's been years since I wrote a module; I thought there was a reason the bit variable was a package variable.

Reply:
I brought this up at the hackathon at the TPRC.  The bottom line is that we couldn't figure out why these things need to be package variables, but they all are.  Changing this one alone would make it the single exception.   So, it's better to leave it as-is.  

A future effort could be to change them all at once.  When I first joined the project I was horrified that we had these magic values that relied on manual synchronization between C constants and perl code.  But it has worked.  It just occurred to me that Configure could be extended to furnish values to both.

So, I will merge this PR by July 3 unless there is protest